src/esthri/src/main.rs

```diff
@@ -26,6 +30,8 @@ use structopt::StructOpt;
 use hyper::Client;
 use tokio::runtime::Builder;
 
+const REAL_AWS_EXECUTABLE: &str = "aws.real";
```

Comment:
I wonder if it would be better to have / expect the user to store the full path to this binary as an environment variable. Could also throw in some checks if the binary is not found at the specified path location.

Reply:
Good idea, an env var that falls back to `aws.real` if it's not specified is probably the best of both worlds here.